gen/js/chromestatus-openapi/src/models/CreateOriginTrialRequest.ts

```diff
@@ -140,6 +140,12 @@ export interface CreateOriginTrialRequest {
      * @memberof CreateOriginTrialRequest
      */
     ot_action_requested?: FieldInfo;
+    /**
+     * 
+     * @type {FieldInfo}
+     * @memberof CreateOriginTrialRequest
+     */
+    ot_creation__bypass_file_checks?: FieldInfo;
```

Comment:
Are these generated files?

Reply:
Yes, these are generated with the `npm run openapi` command